app/templates/macros/helpers.html

```diff
@@ -11,3 +11,11 @@
   } %}
   {{attr|xmlattr}}
 {%- endmacro -%}
+
+{%- macro telephone_number(aria_describedby=None, class=None) -%}
+  {% set attr = {
+    "aria-describedby": aria_describedby,
+    "class": class
+  } %}
+<a href="tel:03001234931" {{attr|xmlattr}}">0300 1234 931</a>
```

Comment:
There is an unnecessary `"` after the `}}` - on the contact-us page the source html is`<a href="tel:03001234931"  aria-describedby=""">0300 1234 931</a></p>`

Reply:
Yes I think that's the empty declaration, now removed

Reply:
No, it carries through, will check

Reply:
Stray quote in the macro. Now fixed